WORKSPACE

```diff
@@ -18,6 +18,22 @@ load("@bazel_skylib//lib:versions.bzl", "versions")
 # in our .travis.yml config.
 versions.check(minimum_bazel_version = "0.22.0")
 
+http_archive(
+    name = "io_bazel_rules_webtesting",
+    sha256 = "f89ca8e91ac53b3c61da356c685bf03e927f23b97b086cc593db8edc088c143f",
+    urls = [
+        # tag 0.3.1 resolves to commit afa8c4435ed8fd832046dab807ef998a26779ecb (2019-04-03 14:10:32 -0700)
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_webtesting/releases/download/0.3.1/rules_webtesting.tar.gz",
+        "https://github.com/bazelbuild/rules_webtesting/releases/download/0.3.1/rules_webtesting.tar.gz",
+    ],
+)
+
+load("@io_bazel_rules_webtesting//web:repositories.bzl", "web_test_repositories")
+web_test_repositories()
```

Comment:
Hmm I think I am slowly getting this. Just wondering, it looks like there is some logic to load skylib inside.
https://github.com/bazelbuild/rules_webtesting/blob/14137ff84f4b2015ed2f815c59df0c1ef9a227e0/web/repositories.bzl#L42-L43

Do we define skylib dependency only because we use versions in it?

Reply:
Yes. Removing the `bazel_skylib` archive and the `versions` check yields
a working state. Keeping `bazel_skylib` at a version distinct from that
of `rules_webtesting` causes problems.

It is also technically sufficient to fetch `@bazel_skylib` only
indirectly through `rules_webtesting`, due to the definition that you
link to. But this seems like a bad idea (include-what-you-use and all).